src/Codeception/Step.php

```diff
@@ -169,13 +201,16 @@ protected function stringifyArgument($argument)
             }
         }
         $arg_str = json_encode($argument, JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES);
-        $arg_str = str_replace('\"', '"', $arg_str);
-        return $arg_str;
+        return str_replace('\"', '"', $arg_str);
     }
 
-    protected function getClassName($argument)
+    /**
+     * @param Closure|MockObject $argument
```

Comment:
This type is incorrect, because this method can receive instance of any class, but only 2 special cases are mentioned.